change/@uifabric-example-app-base-2020-01-15-14-08-56-messageBarHighContrastLinks.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "MessageBar: Fix High Contrast Mode issues.",
+  "packageName": "@uifabric/example-app-base",
```

Comment:
i dont think there is any change in this package?

Reply:
There was before the latest changes, but afterwards there are not anymore.